src/drive/targets/mobile/InitAppMobile.jsx

```diff
@@ -173,6 +174,18 @@ class InitAppMobile {
         text: t('mobile.notifications.backup_paused')
       })
     } */
+    const currentState = store.getState()
```

Comment:
Can we add a comment explaining that onPause is in fact also called before the app is closed?